src/epub.rs

```diff
@@ -27,12 +27,58 @@ pub enum EpubVersion {
     V30,
 }
 
+/// The page-progression-direction attribute of spine is a global attribute and
+/// therefore defines the pagination flow of the book as a whole.
+#[derive(Debug, Copy, Clone, Default)]
+pub enum PageDirection {
+    /// Left to right
+    #[default]
+    Ltr,
+    /// Right to left
+    Rtl,
+}
+
+impl ToString for PageDirection {
+    fn to_string(&self) -> String {
+        match &self {
+            PageDirection::Rtl => "rtl".into(),
+            PageDirection::Ltr => "ltr".into(),
+        }
+    }
+}
+
+impl<'a> From<&'a str> for PageDirection {
+    fn from(value: &'a str) -> Self {
+        value.parse().unwrap_or(PageDirection::default())
+    }
+}
+
+impl From<String> for PageDirection {
+    fn from(value: String) -> Self {
+        value.parse().unwrap_or(PageDirection::default())
+    }
+}
```

Comment:
Why do we need instances that silently turn invalid strings into default values?